internal/initialize/initial.go

```diff
@@ -34,6 +35,9 @@ func init() {
 	fsSum.String("algorithm", algorithm, "hashing algorithm for hashing data")
 	fsSum.String("cm-name", configMapNameForHasher, "name of configMap for hasher")
 	fsSum.String("main-process-name", mainProcessName, "the name of the main process to be monitored by the hasher")
+	fsSum.String("process", procToMonitor, "the name of the process to be monitored by the hasher")
+	fsSum.String("monitoring-path", pathToMonitor, "the service path to be monitored by the hasher")
```

Comment:
Naming - ?
- "main-process-name" - it's a key to search, right? If so, the usage string doesn't match. 
- "process", "monitoring-path" - are not consistent. Maybe, "monitoring-process", "monitoring-path"?

Reply:
> "main-process-name" - it's a key to search, right? If so, the usage string doesn't match.  
Purposes of this flag `main-process-name` is not clear, it seems, it ain't useful at all. Anyway, monitoring process name is passed via `process` flag.

> "process", "monitoring-path" - are not consistent. Maybe, "monitoring-process", "monitoring-path"?
Longer name are not always more descriptive, so I think that current flag names "process", "monitoring-path" are good fit for this context.